src/components/exercises/LetterRotated.js

```diff
@@ -9,29 +9,41 @@ class LettersRotated extends Component {
     super(props);
 
     this.state = {
-      highlighted: null
+      highlighted: [false, false, false, false, false]
```

Comment:
This won't work for the difficulty levels where more than 5 letters are used. You could create this dynamically instead:

```.js
this.state = {
  highlighted = props.letters.map(() => false)
};
```